LocalSettings.php

```diff
@@ -2656,6 +2656,13 @@
 		'default' => [ 'png' ],
 	],
 	// Namespaces [MWNamespaces]
+		'wgExtraSignatureNamespaces' => [
+		'default' => [],
```

Comment:
Please make this:

```
'wgExtraSignatureNamespaces' => [
	'default' => [],
	'nonciclopediawiki' => [
		NS_HELP,
		NS_PROJECT,
	],
],
```

Also move this to some where under '// Namespaces [MWNamespaces]' or in that section please.